js/ui/html_editor/ui/formDialog.js

```diff
@@ -66,7 +66,7 @@ class FormDialog {
             deferRendering: false,
             focusStateEnabled: false,
             showCloseButton: false,
-            wrapperAttr: { class: 'dx-formdialog' },
+            wrapperAttr: { class: DIALOG_CLASS },
```

Comment:
why do we still use `wrapperAttr` here if it's a reason of initial problem?

Reply:
https://github.com/DevExpress/DevExtreme/pull/22834/commits/14aaa27b61571215ec40b31e0ba8fac2741eb32b